gammapy/datasets/flux_points.py

```diff
@@ -259,7 +261,13 @@ def flux_pred(self):
         """Compute predicted flux."""
         flux = 0.0
         for model in self.models:
-            flux += model.spectral_model(self.data.energy_ref)
+            flux_model = model.spectral_model(self.data.energy_ref)
+            if model.temporal_model is not None:
+                integral = model.temporal_model.integral(
+                    self.gti.time_start, self.gti.time_stop
```

Comment:
I think `FluxPointsDataset` currently does not have a `.gti` attribute. However the `FluxMaps` and `FluxPoints` have a `.gti` attribute which, could be re-used. We should think about which way to go and how to serialize the GTI information as well.

Reply:
the `self.gti` property returns `self.data.gti`

Reply:
We are basically re-using the `FluxPoints.gti here`. Should it go on `init` ?

Reply:
Sorry, I didn’t realize FluxPointsDataset.gti already existed. I has been added in an earlier PR. I think the current behavior is fine.